test.js

```diff
@@ -45,6 +45,16 @@ test('spawn cli and not show version', async t => {
 	t.is(stdout, 'version\nautoVersion\nmeow\ncamelCaseOption');
 });
 
+test('spwan cli test input without version output', async t => {
+	const {stdout} = await execa('./fixture.js', ['bar', '--version']);
```

Comment:
You don't need to spawn for this kind of test. Better to just use the `meow` `argv` option. See other tests.